src/OptiHPLCHandler/empower_instrument_method.py

```diff
@@ -273,7 +275,7 @@ def instrument_method_factory(method_definition: Mapping[str, str]) -> Instrumen
         if method_definition["name"] in ["rAcquityFTN"]:
             logger.debug("Creating SampleManager")
             return SampleManagerMethod(method_definition)
-        elif method_definition["name"] in ["AcquityBSM"]:
+        elif method_definition["name"] in ["AcquityBSM", "ACQ-BSM"]:
```

Comment:
cant wait to see how long these lists get...

Reply:
We could to a 
```
if "SBM" in method_definition["name"]
```
instead. That will be broader, but also more risky. Let's discuss.